packages/plotkicadsch/plotkicadsch.0.5.2/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
+   
+maintainer: "Jean-Noel Avila <jn.avila@free.fr>"
+   
+homepage: "https://jnavila.github.io/plotkicadsch/"
+   
+bug-reports: "https://github.com/jnavila/plotkicadsch/issues"
+   
+doc: "https://jnavila.github.io/plotkicadsch/index"
+   
+synopsis: "Utilities to print and compare version of Kicad schematics"
+   
+description: """
+Two utilities:
+ * plotkicadsch is able to plot schematic sheets to SVG files
+ * plotgitsch is able to compare git revisions of schematics
+"""
+   
+license: "ISC"
+   
+dev-repo: "git+https://github.com/jnavila/plotkicadsch.git"
+   
+build: [
+  [ "dune" "subst" ] {pinned}
+    [ "dune" "build" "-p" name ]
+]
+   
+depends: [
+  "dune" {build}
+    "kicadsch" {= "0.5.2"}
+    "tyxml" {>= "4.0.0"}
```

Comment:
could you unindent the next lines please, I'm not sure why everything is indented one way but not the first one. Indentation usually is 2.